homeassistant/components/scene/hunterdouglas_powerview.py

```diff
@@ -11,16 +11,17 @@
 
 _LOGGER = logging.getLogger(__name__)
 REQUIREMENTS = [
-    'https://github.com/sander76/powerviewApi/'
-    'archive/cc6f75dd39160d4aaf46cb2ed9220136b924bcb4.zip#powerviewApi==0.2']
+    'https://github.com/sander76/powerviewApi/archive'
+    '/246e782d60d5c0addcc98d7899a0186f9d5640b0.zip'
```

Comment:
You are missing something like `#powerviewApi=0.2` at the end of the requirements line.